completion/tcolorbox.cwl

```diff
@@ -128,13 +128,29 @@ all
 \tcbsetforeverylayer{%<options%>}
 \tcbox[%<options%>]{%<box content%>}
 \newtcolorbox{envname}{options%keyvals}#N
-\newtcolorbox[init options%keyvals]{envname}[args][default]{options%keyvals}#*N
+\newtcolorbox{envname}[args]{options%keyvals}#N
+\newtcolorbox{envname}[args][default]{options%keyvals}#N
+\newtcolorbox[init options]{envname}{options%keyvals}#N
+\newtcolorbox[init options]{envname}[args]{options%keyvals}#N
+\newtcolorbox[init options]{envname}[args][default]{options%keyvals}#N
```

Comment:
For example, mark all but
```tex
\newtcolorbox{envname}{options%keyvals}#N
\newtcolorbox[init options]{envname}[args][default]{options%keyvals}#N
```
as `#*N`. This will show only two lines in completion list for `\newtcolorbox`, and recognize all six variant usages.

Reply:
correct